homeassistant/components/govee_light_api/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""The Govee Lights - Local API integration."""
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import GoveeLocalApiCoordinator
+
+PLATFORMS: list[Platform] = [Platform.LIGHT]
+SCAN_INTERVAL = timedelta(seconds=30)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Govee Local API from a config entry."""
+
+    coordinator: GoveeLocalApiCoordinator = GoveeLocalApiCoordinator(
+        hass=hass, scan_interval=SCAN_INTERVAL, logger=_LOGGER
```

Comment:
Again, these values are static, no need to pass them to the coordinator. If you disagree with something please respond instead of ignoring it.

Reply:
Apologies, I didn't mean to ignore the comment. I guess I forgot to remove them all form the constructor, but I agree, there is no need to pass them to the constructor.